.rubocop_todo.yml

```diff
@@ -187,20 +187,6 @@ RSpec/PredicateMatcher:
     - 'python/spec/dependabot/python/file_parser/pyproject_files_parser_spec.rb'
     - 'updater/spec/dependabot/job_spec.rb'
 
-# Offense count: 35
```

Comment:
do we know why this was disabled and will there be any issue in enabling it based on the comment that comment at line no:191 ?